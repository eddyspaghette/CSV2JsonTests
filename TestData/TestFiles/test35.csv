1!100!200!300!400!500
2!101!201!301!401!501
3!102!202!302!402!502
4!103!203!303!403!503
5!104!204!304!404!504
6!105!205!305!405!505
7!106!206!306!406!506
8!107!207!307!407!507
9!108!208!308!408!508
10!109!209!309!409!509
11!110!210!310!410!510
12!111!211!311!411!511
13!112!212!312!412!512
14!113!213!313!413!513
15!114!214!314!414!514
22!121!221!321!421!521
23!122!222!322!422!522
18!117!217!317!417!517
19!118!218!318!418!518
20!119!219!319!419!519
21!120!220!320!420!520
22!121!221!321!421!521
23!122!222!322!422!522
24!123!223!323!423!523
25!124!224!324!424!524
2!101!201!301!401!501
