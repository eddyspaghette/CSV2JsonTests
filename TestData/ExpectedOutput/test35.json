[
 {
   "1!100!200!300!400!500": "2!101!201!301!401!501"
 },
 {
   "1!100!200!300!400!500": "3!102!202!302!402!502"
 },
 {
   "1!100!200!300!400!500": "4!103!203!303!403!503"
 },
 {
   "1!100!200!300!400!500": "5!104!204!304!404!504"
 },
 {
   "1!100!200!300!400!500": "6!105!205!305!405!505"
 },
 {
   "1!100!200!300!400!500": "7!106!206!306!406!506"
 },
 {
   "1!100!200!300!400!500": "8!107!207!307!407!507"
 },
 {
   "1!100!200!300!400!500": "9!108!208!308!408!508"
 },
 {
   "1!100!200!300!400!500": "10!109!209!309!409!509"
 },
 {
   "1!100!200!300!400!500": "11!110!210!310!410!510"
 },
 {
   "1!100!200!300!400!500": "12!111!211!311!411!511"
 },
 {
   "1!100!200!300!400!500": "13!112!212!312!412!512"
 },
 {
   "1!100!200!300!400!500": "14!113!213!313!413!513"
 },
 {
   "1!100!200!300!400!500": "15!114!214!314!414!514"
 },
 {
   "1!100!200!300!400!500": "22!121!221!321!421!521"
 },
 {
   "1!100!200!300!400!500": "23!122!222!322!422!522"
 },
 {
   "1!100!200!300!400!500": "18!117!217!317!417!517"
 },
 {
   "1!100!200!300!400!500": "19!118!218!318!418!518"
 },
 {
   "1!100!200!300!400!500": "20!119!219!319!419!519"
 },
 {
   "1!100!200!300!400!500": "21!120!220!320!420!520"
 },
 {
   "1!100!200!300!400!500": "22!121!221!321!421!521"
 },
 {
   "1!100!200!300!400!500": "23!122!222!322!422!522"
 },
 {
   "1!100!200!300!400!500": "24!123!223!323!423!523"
 },
 {
   "1!100!200!300!400!500": "25!124!224!324!424!524"
 },
 {
   "1!100!200!300!400!500": "2!101!201!301!401!501"
 }
]